03-hello-GD-For-Revit/03-05_refinery-toolkit/03-05-02_using-the-refinery-toolkit.md

```diff
@@ -0,0 +1,54 @@
+# Using the Refinery Toolkit
+The package comes with a number of sample files provided to help you get started with each of the libraries.
+
+It is highly recommended to start with the samples as they contain detailed notes and instructions on how to use each of the nodes. Feel free to open an issue or submit a PR on the [GitHub repo](https://github.com/DynamoDS/RefineryToolkits) if you’d like to see further documentation added. 
+
+You can find the sample files in the package repository’s samples folder, as well as in the extra folder of the package when downloaded through the Dynamo Package Manager, typically found here: `%appdata%\Dynamo\Dynamo Revit\2\packages\Refinery Toolkit`
+
+The samples show simple examples of how each node in the toolkit works. Some of the samples have a Revit version (marked with `(RVT)`) and some also have a version to be used directly with Generative Design in Revit (marked with `(RefineryVersion)`).
+
```

Comment:
What do you mean by "Refinery Version" ?

Reply:
Hmm, that is something weird with the samples and should probably be removed. I think the idea was to indicate something that worked with `Core` vs `Revit` context. 
![image](https://user-images.githubusercontent.com/15744724/106185081-9a184000-615f-11eb-8686-a4bb1ae94988.png)